nep-12.mediawiki

```diff
@@ -0,0 +1,2130 @@
+  "script": {
+    "invocation": "402a1dab9e5593d1d7d2a22a36772d4541b8053d33f8b8474b7d5a20066c1bd821e051fc252ed16146930d55ecb17fbb74972fba4c4b27af81a707999ca1313dd2401520eba2dd3b54a74a798cbb716c484ba6f6f21218f099e3d622a0fbd15989f38f9b0b344daf9b89175055d3a92f49df65118e8598735d651bedd4f1811baeb140e6491c03f3057f404d2fe7db50e40e82ade405a9dc7fccd81f4ba0b499a4a29f8570d631b8d40c5995b17d9391fe9ff8c73f28a4e1eb922b7a1ce9d1a5dc0448402cfcdede54828875d45402120aa2d8f78c7bd40df5e5d3b1873fd7e4d03672ebd0904f90c90fa519c623968f55550ae55374de66dc0db9c9d865c593bb95be5640214db0cd3cea6f4ad866df4129d482b89583805d1bdb08ce8399881e70351778a3e4a4093cf69aa7b99b83347fbfd38d85ff45d6a78ca2ab8cacffbfbc8c2d16",
+    "verification": "5521030ef96257401b803da5dd201233e2be828795672b775dd674d69df83f7aec1e36210327da12b5c40200e9f65569476bbff2218da4f32548ff43b6387ec1416a231ee821025bdf3f181f53e9696227843950deb72dcd374ded17c057159513c3d0abe20b64210266b588e350ab63b850e55dbfed0feeda44410a30966341b371014b803a15af0721026ce35b29147ad09e4afe4ec4a7319095f08198fa8babbe3c56e970b143528d222103c089d7122b840a4935234e82e26ae5efd0c2acb627239dc9f207311337b6f2c12103fd95a9cb3098e6447d0de9f76cc97fd5e36830f9c7044457c15a0e81316bf28f57ae"
+  },
+  "tx": [
+    {
+      "txid": "0x07d70f7337d3869a7daa538425d78a47212fb8c6130d66d84ac48526853a4e51",
+      "size": 10,
+      "type": "MinerTransaction",
+      "version": 0,
+      "attributes": [],
+      "vin": [],
+      "vout": [],
+      "sys_fee": "0",
+      "net_fee": "0",
+      "scripts": [],
+      "nonce": 3093227147
+    }
+  ],
+  "confirmations": 70,
+  "nextblockhash": "0x2c9d6a107b21e83e09dd1b89df344a726895147d410120c46996290692ba29aa"
+}
+</pre>
+
+
+====getTransaction====
+Get information about a specific transaction.
+
+
+'''Method Interface'''
+
+<pre>
+function getTransaction({
+  txid: string,
+  network?: string
+}): Promise<{result: TransactionDetails}>
+</pre>
+
+
+'''Input arguments'''
+
+<pre>
+interface TransactionInputArgs {
+  txid: string;
+  network?: string;
+}
+</pre>
+
+
+'''Success return value'''
+
+<pre>
+export interface TransactionDetails {
+  txid: string;
+  size: number;
+  type: string;
+  version: number;
+  attributes: TransactionAttribute[];
+  vin: any[];
+  vout: any[];
+  sys_fee: string;
+  net_fee: string;
+  scripts: TransactionScript[];
+  script: string;
+  gas: string;
+  blockhash: string;
+  confirmations: number;
+  blocktime: number;
+}
+
+interface TransactionAttribute {
+  usage: string;
+  data: string;
+}
+
+interface TransactionScript {
+  invocation: string;
+  verification: string;
+}
+</pre>
+
+
+'''Error return value'''
+
+<pre>
+interface Error {
+  type: string;
+  description: string;
+  data: string;
+}
+</pre>
+
+
+'''Example'''
+
+<pre>
+getTransaction({
+  txid: '7e049fd7c253dabf38e4156df30c78b30d49f307196aa89b99a47d2330789bf2',
+  network: 'TestNet'
+})
+.then((result: Object) => {
+  console.log('Transaction details: ' + JSON.stringigy(result));
```

Comment:
typo on `stringify`. I don't know if it's good to have complete JS code here on specification...

Reply:
Nice catch! Updated.